index.bs

```diff
@@ -1076,8 +1076,8 @@ run the following steps:
 1. If |name|'s [=string/length=] is 0:
     1. If |value| contains U+003D (=), then return failure.
     1. If |value|'s [=string/length=] is 0, then return failure.
-    1. If |value|, [=byte-lowercased=], [=byte sequence/starts with=] \``__host-`\`, \``__hosthttp-`\`, \``__http-`\`, or \``__secure-`\`, then return failure.
-1. If |name|, [=byte-lowercased=], [=byte sequence/starts with=] \``__http-`\` or \``__hosthttp-`\`, then return failure.
+    1. If |value|, [=byte-lowercased=], [=byte sequence/starts with=] \``__host-`\`, \``__host-http-`\`, \``__http-`\`, or \``__secure-`\`, then return failure.
+1. If |name|, [=byte-lowercased=], [=byte sequence/starts with=] \``__http-`\` or \``__host-http-`\`, then return failure.
```

Comment:
We should reverse the order of __http and __host-http so it's consistent with the list above (which is lexicographical if I'm not mistaken).